A32NX/html_ui/Pages/A32NX_Core/A32NX_TransitionAltitude.js

```diff
@@ -0,0 +1,346 @@
+    { icao: "EN", transAlt: 7000 },
+    { icao: "EP", transAlt: 6500 },
+    { icao: "ES", transAlt: 9000 },
+    { icao: "ET", transAlt: 5000 },
+    { icao: "EV", transAlt: 5000 },
+    { icao: "EY", transAlt: 5000 },
+    { icao: "FA", transAlt: 9500 },
+    { icao: "FB", transAlt: 7000 },
+    { icao: "FC", transAlt: 6000 },
+    { icao: "FD", transAlt: 8000 },
+    { icao: "FE", transAlt: 4200 },
+    { icao: "FG", transAlt: 6000 },
+    { icao: "FH", transAlt: 6000 },
+    { icao: "FI", transAlt: 4000 },
+    { icao: "FJ", transAlt: 17000 },
+    { icao: "FK", transAlt: 6300 },
+    { icao: "FL", transAlt: 7000 },
+    { icao: "FM", transAlt: 12000 },
+    { icao: "FN", transAlt: 10000 },
+    { icao: "FO", transAlt: 4500 },
+    { icao: "FP", transAlt: 4500 },
+    { icao: "FQ", transAlt: 7500 },
+    { icao: "FS", transAlt: 4500 },
+    { icao: "FT", transAlt: 4400 },
+    { icao: "FV", transAlt: 8000 },
+    { icao: "FW", transAlt: 7000 },
+    { icao: "FX", transAlt: 12000 },
+    { icao: "FY", transAlt: 10000 },
+    { icao: "FZ", transAlt: 14000 },
+    { icao: "GA", transAlt: 4000 },
+    { icao: "GB", transAlt: 2500 },
+    { icao: "GC", transAlt: 6000 },
+    { icao: "GE", transAlt: 6000 },
+    { icao: "GF", transAlt: 5000 },
+    { icao: "GG", transAlt: 3000 },
+    { icao: "GL", transAlt: 3000 },
+    { icao: "GM", transAlt: 10000 },
+    { icao: "GO", transAlt: 3500 },
+    { icao: "GQ", transAlt: 3100 },
+    { icao: "GU", transAlt: 6000 },
+    { icao: "GV", transAlt: 7000 },
+    { icao: "HA", transAlt: 14000 },
+    { icao: "HB", transAlt: 9500 },
+    { icao: "HD", transAlt: 5000 },
+    { icao: "HR", transAlt: 9000 },
+    { icao: "HU", transAlt: 7000 },
+    { icao: "LA", transAlt: 10000 },
+    { icao: "LD", transAlt: 9500 },
+    { icao: "LH", transAlt: 10000 },
+    { icao: "LJ", transAlt: 10500 },
+    { icao: "LK", transAlt: 5000 },
+    { icao: "LL", transAlt: 18000 },
+    { icao: "LM", transAlt: 5000 },
+    { icao: "LO", transAlt: 10000 },
+    { icao: "LQ", transAlt: 9500 },
+    { icao: "LU", transAlt: 4000 },
+    { icao: "LW", transAlt: 11000 },
+    { icao: "LX", transAlt: 6000 },
+    { icao: "LY", transAlt: 10000 },
+    { icao: "LZ", transAlt: 10000 },
+    { icao: "MB", transAlt: 6000 },
+    { icao: "MD", transAlt: 17000 },
+    { icao: "MG", transAlt: 19000 },
+    { icao: "MK", transAlt: 17000 },
+    { icao: "MM", transAlt: 18500 },
+    { icao: "MN", transAlt: 19500 },
+    { icao: "MO", transAlt: 18000 },
+    { icao: "MP", transAlt: 18000 },
+    { icao: "MR", transAlt: 19000 },
+    { icao: "MT", transAlt: 17000 },
+    { icao: "MW", transAlt: 17000 },
+    { icao: "MY", transAlt: 18000 },
+    { icao: "MZ", transAlt: 19500 },
+    { icao: "NC", transAlt: 13000 },
+    { icao: "NG", transAlt: 11000 },
+    { icao: "NI", transAlt: 11000 },
+    { icao: "NS", transAlt: 13000 },
+    { icao: "NV", transAlt: 11000 },
+    { icao: "NW", transAlt: 11000 },
+    { icao: "NZ", transAlt: 18000 },
```

Comment:
Should be 13000 on the way up and FL150 on the way down.

Reply:
can you fix it? you mean TA is 13,000 TL is FL150?

it is for offline, can't provide large db :(
Also there is no level logic yet. i'll add that on other PR

Reply:
Yeah, make NZ 13000 TA if you can (that is correct for the whole country, which is covered by ICAO code NZ).

Reply:
OK!